packages/config/src/VitessceAutoConfig.js

```diff
@@ -418,7 +555,24 @@ async function generateConfig(url, vc) {
   return views;
 }
 
-export async function generateConfigs(fileUrls) {
+export function getDatasetType(fileUrls) {
+  const fileTypes = {};
+
+  fileUrls.forEach((url) => {
+    const match = getFileType(url);
+    // hints config only has OME-TIFF, since settings are the same for both OME-TIFF and OME-Zarr
+    if (match.name === 'OME-Zarr') {
+      fileTypes['OME-TIFF'] = true;
+    } else {
+      fileTypes[match.name] = true;
+    }
+  });
+
+  return Object.keys(fileTypes);
+}
+
+
+export async function generateConfigs(fileUrls, hintsConfig, hintsType, useHints) {
```

Comment:
```suggestion
export async function generateConfig(fileUrls, hintsConfig, hintsType, useHints) {
```

I think we should rename this to the singular `generateConfig` since it is only generating one view config.